src/PoolManager.sol

```diff
@@ -144,7 +145,7 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
     function _accountDelta(Currency currency, int128 delta) internal {
         if (delta == 0) return;
 
-        int256 current = currencyDelta[msg.sender][currency];
+        int256 current = msg.sender.getCurrencyDelta(currency);
```

Comment:
this syntax is kinda funky but I do like it as a library 🤷‍♀️

Reply:
Yeah i wasnt 100% on it either. Would you prefer it as
`CurrencyDelta.get(msg.sender, currency)` and `CurrencyDelta.set(msg.sender, currency, delta)`
or
`msg.sender.getCurrencyDelta(currency)` and `msg.sender.setCurrencyDelta(currency, delta)`

Reply:
took a poll and went with the first one!